src/bp-forums/common/functions.php

```diff
@@ -1049,6 +1049,17 @@ function bbp_check_for_blacklist( $anonymous_data = false, $author_id = 0, $titl
 
 /** Subscriptions *************************************************************/
 
+/**
+ * Allow HTML tags for the content of the forum and its reply, when sending email notifications to subscribers
+ *
+ * @return array HTML tags
+ */
+function bbp_get_forum_allow_tags() {
```

Comment:
@milanhirapra If this tab list only for email notification then function name should be `bbp_get_forum_email_notifications_allow_tages`

Reply:
@dipskakadiya Renamed the function name as per your suggestion.